### PR TITLE
More generic SerializersRegistrar API in place of SerializersProvider…

### PR DESCRIPTION
**Pull request by arkadius**
…, moved AvroSerializersRegistrar to avro utils as a service

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -544,7 +544,7 @@ lazy val avroFlinkUtil = (project in engine("flink/avro-util")).
       )
     }
   )
-  .dependsOn(kafkaFlinkUtil, kafkaTestUtil % "test", flinkTestUtil % "test", process % "test")
```
Comment (mproch) [inline build.sbt:547]: maybe we can have SerializersRegistrar in flink-api?

**Review by gadomsky — APPROVED** [review]
LGTM

**Review by mproch — APPROVED** [review]
:+1: I added one minor coment

File: build.sbt (hunk 1 above)
Comment (arkadius) [reply build.sbt]: fixed

**Pull request merged**